tests/util.h

```diff
@@ -117,9 +117,30 @@ make_lengths()
 }
 
 // Only used in examples. Do not use otherwise.
-void generate_random(uint8_t* dst, size_t dst_len) {
-  EverCrypt_DRBG_state_s_s *state = EverCrypt_DRBG_create(Spec_Hash_Definitions_SHA2_256);
-  EverCrypt_DRBG_instantiate(state, (uint8_t*) "example", 7);
-  EverCrypt_DRBG_generate(dst, state, dst_len, (uint8_t*)"", 0);
+void
+generate_random(uint8_t* output, uint32_t output_len)
+{
+  EverCrypt_DRBG_state_s_s* state =
+    EverCrypt_DRBG_create(Spec_Hash_Definitions_SHA2_256);
+  EverCrypt_DRBG_instantiate(state, (uint8_t*)"example", 7);
+  EverCrypt_DRBG_generate(output, state, output_len, (uint8_t*)"", 0);
   EverCrypt_DRBG_uninstantiate(state);
 }
+
+// ANCHOR(print_hex_ln)
+void
+print_hex_ln(size_t bytes_len, uint8_t* bytes)
+{
+  for (int i = 0; i < bytes_len; ++i) {
+    printf("%02x", bytes[i]);
+  }
+
+  printf("\n");
+}
+// ANCHOR_END(print_hex_ln)
+
+string
+test_vector_path(string path)
+{
+  return "./../../assets/test_vectors/" + path;
```

Comment:
In a previous version I used [C++17's filesystem library](https://en.cppreference.com/w/cpp/filesystem). However, this is not implemented in GCC7, see https://github.com/cryspen/hacl-packages/actions/runs/3436483986/jobs/5730054053 run. We could use `experimental/filesystem` but it makes problems on macOS. Long story short: The path separators work in our CI and, given that this is only test code, we might get away with it as soon as noone files an issue.

Reply:
The reason I am doing this is that copying test vectors to a new location before reading them is totally confusing.

* You update the test vectors and change the source -> New test vector is read
* You update the test vectors again (and do not change the source) -> Old test vector is read

I am pretty sure that the first time someone starts to work on the test vectors this will bite them. (It has bitten me at least twice.)

Reply:
I'll move these changes to another PR.